src/domain/animal_detection_mode.py

```diff
@@ -114,6 +122,12 @@ def check_for_termination_requests(self):
             for camera in cameras:
                 if camera.type == Camera.CameraTypes.USB_CAMERA:
                     camera.stop_thread = True
+
+            # Stop HTTPS Actuator Server
```

Comment:
As also the start update actuator threads is in base class, also this logic should be moved in base class